binaries/aot_model_compiler.cc

```diff
@@ -217,27 +112,6 @@ int main(int argc, char** argv) {
   auto m = torch::jit::load(FLAGS_model);
   m.eval();
   auto frozen_m = torch::jit::freeze_module(m.clone());
-  auto graph = frozen_m.get_method(FLAGS_method_name).graph();
-  auto inputShapes = parseInputShapes();
-  auto inputTypes = parseInputTypes();
-  std::vector<c10::optional<at::Tensor>> example_inputs;
-  example_inputs.reserve(inputShapes.size());
-  for (int i = 0; i < inputShapes.size(); ++i) {
-    example_inputs.emplace_back(at::rand(inputShapes[i]).to(at::dtype(inputTypes[i])));
-  }
-
-  torch::jit::RemoveTensorMutation(graph);
```

Comment:
Can we also remove the corresponding headers imported for this and others?

Reply:
Good point, thanks!